src/components/Popups/PopupBase.tsx

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
 import classNames from 'classnames';
-import createPopper, { Instance, Options } from './Popper';
+import createPopper, { Instance, Options, VirtualElement } from './Popper';
 import './PopupBase.scss';
 
+export type PopupReference = Element | VirtualElement;
```

Comment:
Let's declare this in src/components/Popper, since it's used there too. You can re-export it here if you want.